### PR TITLE
Update package author to Brigade Group, Inc.

### PR DESCRIPTION
**Pull request by MatthewHerbst**
This updates the package author to be the company.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
     "eslint",
     "config"
   ],
-  "author": "Joe Lencioni",
+  "author": "Brigade Group, Inc.",
```
Comment (lencioni) [inline package.json:17]: Brigade Group? I thought it was Brigade Media. http://www.brigadegroup.com/

Comment (MatthewHerbst) [reply package.json:17]: "Brigade Group, Inc." from the [ToS](https://www.brigade.com/tos) which I would hope is accurate ❓ 😟

Comment (lencioni) [reply package.json:17]: Ah, ok!

Also note that Brigade Media is mentioned in the ToS. :dancing_men:

Comment (MatthewHerbst) [reply package.json:17]: It seems they changed the name a while back from Brigade Media to Brigade Group. I'll bring it up with whoever controls our ToS, thanks!

**Pull request merged**